### PR TITLE
introduce a script that can generate a new component from the cli

### PR DESCRIPTION
**Pull request by walkerrandolphsmith**
I couldn't get yarn to install due to what seems to be a dependency that is not in the public npm registry: 

```
error An unexpected error occurred: “https://registry.yarnpkg.com/@versionone%2fbabel-plugin-react-docgen: Not found”.
```

This pr would require root level packages installed

### CODE REVIEW

**Comment by andrew-codes** [comment]
The install error is due to a package in the root package.json that should be removed. I will have a PR open that does this that will merge soon.

Could we make create a CLI build package? In other words:
- stick the code in a `create` directory within the `build` directory
- give it a proper package.json; including the `"bin": "src/index.js"` field to make it a CLI executable package. If the `build` directory isn't the right term to categorize this, then maybe it could use a rename; utilities or support (packages)

File: create/template/Component.spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+context('PLACEHOLDER', () => {
+  beforeEach(() => {
+    cy.visit('/packages/PLACEHOLDER');
+  });
+
+  specify(
+    'component renders',
+    () => {
+      return cy.get('[data-component="PLACEHOLDER"]');
```
[the remark this answers is not in the document]
Comment (walkerrandolphsmith) [reply create/template/Component.spec.js]: Well it is implemented. When we generate the new component we create a README.mdx which renders the component. Perhaps we create a second test in this file that is throws this.

File: create/template/README.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+import { Playground, PropsTable, ThemeDefinitionTable } from '@versionone/docz';
+import { PLACEHOLDER } from './src';
+
+# PLACEHOLDER
+
+description
+
+<Playground>
```
[the remark this answers is not in the document]
Comment (walkerrandolphsmith) [reply create/template/README.mdx]: 👍

**Comment by walkerrandolphsmith** [comment]
I dig `"bin": "src/index.js"` suggestion and making it an executable package, however putting it in `build` does seem strange since its not related to the `build`. I also think co-locating build and create in a "utils" directory is not necessary. Perhaps I just add the `create` dir to lerna's packages array.

**Comment by andrew-codes** [comment]
We could stick it in the packages directory. That seems the most appropriate place to me.

**Pull request closed without merging**